src/main/java/com/smartsheet/api/internal/FolderResourcesImpl.java

```diff
@@ -64,8 +67,16 @@ public FolderResourcesImpl(SmartsheetImpl smartsheet) {
 	 * rather than returning null)
 	 * @throws SmartsheetException the smartsheet exception
 	 */
-	public Folder getFolder(long folderId) throws SmartsheetException {
-		return this.getResource("folder/" + folderId, Folder.class);
+	public Folder getFolder(long folderId, EnumSet<FolderInclusion> includes) throws SmartsheetException {
```

Comment:
Remove checking for null on the includes as Ahmad is changing the generateUrl() to drop null parameters. The code can look like the following.

```
HashMap<String, String> parameters = new HashMap<String, String>();
parameters.put("include", QueryUtil.generateCommaSeparatedList(includes));
return this.getResource(QueryUtil.generateUrl(null, parameters), Folder.class);
```